test/Serilog.ApprovalTests/Serilog.approved.txt

```diff
@@ -409,6 +410,7 @@ namespace Serilog.Events
     {
         public SequenceValue(System.Collections.Generic.IEnumerable<Serilog.Events.LogEventPropertyValue> elements) { }
         public System.Collections.Generic.IReadOnlyList<Serilog.Events.LogEventPropertyValue> Elements { get; }
+        public static Serilog.Events.SequenceValue Empty { get; }
```

Comment:
@nblumhardt I pulled changes from `dev` to fix approval file after merging #1785.